src/main/java/io/ebean/config/ServerConfig.java

```diff
@@ -488,6 +488,11 @@
 
   private ProfilingConfig profilingConfig = new ProfilingConfig();
 
+  /**
+   * Controls the default order by id setting of queries. See {@link Query#orderById(boolean)}
+   */
+  private boolean defaultOrderById = false;
```

Comment:
Maybe this should default to true ... The exception case is then to turn it off.